messaging/topic_mgt.go

```diff
@@ -25,9 +25,9 @@ import (
 )
 
 const (
-	iidEndpoint    = "https://iid.googleapis.com/iid/v1"
-	iidSubscribe   = ":batchAdd"
-	iidUnsubscribe = ":batchRemove"
+	iidEndpoint    = "https://iid.googleapis.com/iid"
```

Comment:
Instead of this, let's change how the URL is constructed in the `makeTopicManagementRequest()` function.

```
iidSubscribe = "batchAdd"
iidUnsubscribe = "batchRemove"

fmt.Sprintf("%s:%s", c.iidEndpoint, req.op)
```

This will cause a test failure, so we need to update the tests too.